Tweak the example in the explanation of "Lax-allowing-unsafe" (#2812)
httpwg/http-extensions: draft-ietf-httpbis-rfc6265bis.md

cross-site requests with "unsafe" HTTP methods (as was the case prior to the
introduction of the `SameSite` attribute).

> For example, the concluding step of a login flow may involve a cross-site top-level
> `POST` request to an endpoint which expects a cookie with transactional state
> information necessary to securely complete the login. For such a cookie, "Lax"
> enforcement is not appropriate, as it would cause the cookie to be excluded due
> to the unsafe HTTP request method, resulting in an unrecoverable failure of the
> whole login flow.

Reviewer: ```suggestion
For example, the concluding step of a login flow may involve a cross-site top-level
`POST` request to an endpoint; this endpoint expects a recently created cookie
containing transactional state information, necessary to securely complete the
login. For such a cookie, "Lax" enforcement is not appropriate, as it would
cause the cookie to be excluded due to the unsafe HTTP request method,
resulting in an unrecoverable failure of the whole login flow.
```

I think it's a good idea to also motivate the reason for the 2 min limit.

> The "Lax-allowing-unsafe" enforcement mode described in {{lax-allowing-unsafe}}

Reviewer: Seems github won't let me add a suggestion for this text, but I also suggest a change of

```
The "Lax-allowing-unsafe" enforcement mode described in {{lax-allowing-unsafe}}
retains some of the protections of "Lax" enforcement (as compared to "None")
while still allowing recently created cookies to be sent cross-site with unsafe
top-level requests.
```

Author: Yeah, github doesn't let you suggest changes too far from the context of the original changes in the PR. But I'll apply this locally and push and the whole dance. Thanks for the suggestion.

Author: done in 4590b6c0b949791f907eb240c59c3fe0a5a6bace
